src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -327,7 +343,15 @@ public String getText(By by) {
     }
 
     public String getText(WebElement element) {
-        return element.getText();
+        try {
+            return element.getText();
+        } catch (Exception e) {
+            if (e instanceof NoSuchElementException) {
+                throw new NoSuchElementException("Unable to locate element: " + element.toString());
+            } else {
+                throw e;
+            }
+        }
```

Comment:
Is there any way to consolidate these catch lines somewhere since they are reused in each function?

Reply:
Addressed in 5000316 -> consolidated tests to use test annotation instead of try/catch

Reply:
@nerdsosky Ignore me -- for some reason I thought this had to do with the tests around it. I'll take a look at actually implementing your suggestion tomorrow. Sorry for any confusion.

Reply:
No worries, I'm not sure what it would look like implemented or if it's necessary but the repeated code just stuck out to me.

Reply:
@nerdsosky So, after talking to @rf43, I didn't really see any terrible need to reuse the catches, but I did customize them slightly so they are a little unique. (Also I couldn't quite work out an elegant way to accomplish this.)

See my most recent changes though in 569b124

Reply:
Works for me 👍